tests/integration/issues/github_241/test_topk.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import numpy as np
+import pytest
+from jina.flow import Flow
+from jina.proto import jina_pb2
+from jina.drivers.helper import array2pb
+
+
+@pytest.fixture
+def config(tmpdir):
+    os.environ['JINA_TOPK_DIR'] = str(tmpdir)
+    os.environ['JINA_NDOCS'] = '3'
+    os.environ['JINA_TOPK'] = '9'
+    os.environ['JINA_TOPK_OVERRIDE'] = '11'
+
+
+def random_docs(num_docs, embed_dim=10, jitter=1):
+    for j in range(num_docs):
+        d = jina_pb2.Document()
+        d.tags['id'] = j
+        d.text = b'hello'
+        d.embedding.CopyFrom(array2pb(np.random.random([embed_dim + np.random.randint(0, jitter)])))
+        yield d
```

Comment:
this could be a pytest [yield fixture](https://docs.pytest.org/en/2.9.2/yieldfixture.html)